symplyphysics/laws/hydro/pressure_of_liquid_in_vessel_moving_vertically.py

```diff
@@ -0,0 +1,77 @@
+from sympy import (Eq, solve, sqrt)
+from symplyphysics import (
+    Vector,
+    add_cartesian_vectors,
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+    vector_magnitude,
+)
+from symplyphysics.core.expr_comparisons import expr_equals
+from sympy.physics.units import acceleration_due_to_gravity as earth_free_fall_acceleration
+
+from symplyphysics.laws.hydro import hydrostatic_pressure_from_density_and_depth_acceleration as pressure_law
+
+# Description
+## If a vessel with a liquid moves with vertical acceleration, then the hydrostatic pressure of the liquid
+## depends on the density of the liquid, the acceleration of free fall, the vertical acceleration of the vessel
+## and the height of liquid.
+
+## Law is: p = p0 * |g + a| * h, where
+## || - absolute value of the value,
+## p - pressure,
+## p0 - density of liquid,
+## g - earth free fall acceleration,
+## a - acceleration of vessel,
+## h - height.
+
+pressure = Symbol("pressure", units.pressure)
+
+density_liquid = Symbol("density_liquid", units.mass / units.volume)
+acceleration = Symbol("acceleration", units.acceleration)
+height = Symbol("height", units.length)
+
+law = Eq(pressure, density_liquid * sqrt((earth_free_fall_acceleration + acceleration)**2) * height)
```

Comment:
Does not correspond to description. Also now it looks exactly as horizontally accelerated vessel.